.npmrc

```diff
@@ -8,3 +8,4 @@ public-hoist-pattern[]="tsconfig-seek"
 # end managed by skuba
 
 public-hoist-pattern[]="remark-lint-*"
+ignore-workspace-cycles=true
```

Comment:
Silences this:

![image](https://github.com/seek-oss/skuba/assets/18017094/5e09cf56-ea55-4805-8ab9-e0b2bba2cbee)